src/controllers/site.ts

```diff
@@ -77,34 +84,45 @@ const STOPPED_STATES = [GlobalStatus.NotStarted, GlobalStatus.Failed, `STOPPED`]
  * Represents a single user Gatsby site
  */
 export class GatsbySite {
-  runner?: Worker
   siteStatus: ISiteStatus = DEFAULT_STATUS
-  startedInApp?: boolean
+  startedInDesktop?: boolean
 
   private _listeners = new Set<(status: ISiteStatus, action?: Action) => void>()
 
   constructor(
     public root: string,
     public name: string = `Unnamed site`,
+    public hash: string = createContentDigest(root),
     saveMetadata = false
   ) {
     if (saveMetadata) {
       this.saveMetadataToServiceConfig()
     } else {
       this.loadFromServiceConfig()
     }
+    ipcRenderer.on(`site-message`, (event, hash: string, message: IMessage) => {
+      if (hash !== this.hash) {
+        // Not for us
+        return
+      }
```

Comment:
would it be better to support something like `ipcRender.on(this.hash)` and use the hash as the event name?